scripts/windows/condainstall.bat

```diff
@@ -49,3 +39,8 @@ for %%f in ( *.tar.bz2 ) do (
     "%CONDA%" install --yes  --copy --quiet --prefix "%PREFIX%" "%CD%\%%f" ^
         || exit /b !ERRORLEVEL!
 )
+
+rem # Activate the environment makes conda create
+rem # activate.bat, conda.bat and other shortcuts.
+echo Activating the environment for the first time"
+"%ACTIVATE%" "%PREFIX%"
```

Comment:
Which version[s] of conda do this? 4.1.?. Note the comment above about 4.1.* `conda create`.

That the activate, conda scripts are created manually is mostly for compatibility with 4.1.* On later versions `conda create --prefix ...` (without any package name) does create a environment with the scripts (at least if I remember correctly).

We can drop support for installation with 4.1. It is probably no longer relevant.

Reply:
I was using the installer on our download page which includes conda 4.3.14.

I have modified conda create command in installer and tested with the oldest version of conda I could install with python3.6 (4.2.16), and it works. Environment gets created and all the scripts are there. I'll update this PR to drop support for conda 4.1 instead.